sycl/source/detail/platform_impl.cpp

```diff
@@ -20,6 +20,8 @@
 #include <string>
 #include <vector>
 
+#define NumOfBackends (int)backend::all
```

Comment:
Why need this macro at all? Also please use C++ cast.

Reply:
removed it. Thanks